lib/coinbase/user.rb

```diff
@@ -20,21 +20,10 @@ def id
     end
 
     # Creates a new Wallet belonging to the User.
+    # @param network_id [String] (Optional) the ID of the blockchain network. Defaults to 'base-sepolia'.
```

Comment:
Isn't this a hash now? Or do I misunderstand how opts work

Reply:
The arguments are still parameterized such as here:
https://github.com/coinbase/coinbase-sdk-ruby/blob/2b82012c80793d59bacd07ed8b7f5c0cbf596739/lib/coinbase/user.rb#L47-L51

This makes the method signature always stay in sync with `Coinbase::Wallet.create` and use the same set of defaults.